runtime/oti/j9nonbuilder.h

```diff
@@ -108,6 +108,8 @@
 #define J9FieldTypeMask 0x380000
 #define J9FieldTypeShort 0x280000
 
+/* @ddr_namespace: map_to_type=J9RecordComponentFlags */
```

Comment:
Moving the constants below to a new namespace creates a discontinuity in core files.
The change seems reasonable, but I please confirm you did so consciously.

Reply:
I am not familiar with the consequences of this change. What do you recommend to confirm its okay?

Reply:
A core file generated from a VM built before this change would have these constants in `J9FieldFlags`.
I think `J9RecordComponentFlags` is a better place for them, but it will mean `NoSuchFieldError` will be thrown by DDR code that makes use of `J9RecordComponentFlags.J9RecordComponentFlagHasAnnotations` (for example) when accessing such an old core file. Because records are still a preview feature, I can live with that: My point is that it needs to be a conscious decision.

Reply:
Okay. Thanks for the clarification.